src/connectors/ai4europe_cms/ai4europe_cms_event_connector.py

```diff
@@ -0,0 +1,95 @@
+import requests
+
+from requests.exceptions import HTTPError
+from typing import Iterator
+
+from connectors.abstract.resource_connector import ResourceConnector
+from connectors.record_error import RecordError
+from database.model.platform.platform_names import PlatformName
+from database.model.resource_read_and_create import resource_create
+from database.model.event.event import Event
+from connectors.resource_with_relations import ResourceWithRelations
+from database.model.ai_resource.text import Text
+
+
+class AI4EuropeCmsEventConnector(ResourceConnector[Event]):
+    @property
+    def resource_class(self) -> type[Event]:
+        return Event
+
+    @property
+    def platform_name(self) -> PlatformName:
+        return PlatformName.ai4europe_cms
+
+    def run(self, state: dict, **kwargs) -> Iterator[ResourceWithRelations[Event] | RecordError]:
+        """Fetch resources and update the state"""
+
+        url_data = "https://community-dev-api.aiod.eu/api/events/"
+
+        headers = {"AuthorizationToken": "1234567890"}
+
+        response = requests.get(url_data, headers=headers, timeout=600)
+
+        if not response.ok:
+            status_code = response.status_code
+            msg = response.json()["error"]["message"]
+            err_msg = f"Error while fetching {url_data} from AI4Europe CMS: ({status_code}) {msg}"
+            err = HTTPError(err_msg)
+            yield RecordError(identifier=None, error=err)
+            return
+
+        try:
+            events = response.json()
+        except Exception as e:
+            yield RecordError(identifier=None, error=e)
+            return
+
+        for event in events:
+            pydantic_class = resource_create(Event)
+            desc = event.get("description") or {}
+
+            yield ResourceWithRelations[Event](
+                resource=pydantic_class(
+                    platform_resource_identifier=(
+                        event["platform_resource_identifier"]
+                        if event.get("platform_resource_identifier") is not None
+                        else None
+                    ),
+                    platform=event["platform"] if event.get("platform") is not None else None,
```

Comment:
```suggestion
                    platform=event.get("platform")
```
Safely retrieving the value for a key and otherwise returning None is exactly what `dict.get` does, so you can replace this pattern with just a call to `dict.get`.

Reply:
Platform and Platform Identifier are specifically for connectors and are limited to exactly those platforms we have connectors for. In this case I think the platform should simply be `PlatformName.ai4europe_cms` with the identifier of the asset in the ai4europe_cms catalogue.